### ISSUE TITLE
Have the issues mentioned by you been resolved?

### ISSUE THREAD

**Issue by gduzhu** [jonathanio/update-systemd-resolved#114]
Hi Jonathan,

Your helper script, update-systemd-resolved, was released on May 19, 2019. That was more than four years ago.

I am not IT-trained; hence would appreciate it if you could tell me if the following issues mentioned in README.md have been resolved, specifically issues such as:

```
Issues with Ubuntu and Fedora

DNS leakage

Known issues

NetworkManager

DNSSEC issues
```

Thanks for your clarification.

**Comment by redongh** [comment]
Hi gduzhu,

not sure on the exact status but at least it seems that with a current installation of Fedora 38, `nsswitch.conf` still seems to be managed through `authselect` according to the warning header in the .conf-file.

Would love to learn about how to the the stub-resover method to work as I still could not get my openVPN connection to work successfully and suspect that it might be lelated to this.

**Comment by tomeon** [comment]
@gduzhu -- most of the issues mentioned in the README are upstream of `update-systemd-resolved`.  It's likely that at least some of them have been solved (e.g., it looks like the NetworkManager issue was [fixed in NetworkManager release 1.26.6](https://gitlab.freedesktop.org/NetworkManager/NetworkManager/-/blob/nm-1-26/NEWS#L23-24)).

The DNS leakage issue isn't something that `update-systemd-resolved` can solve -- indeed it's not solvable in general.  Instead, it's something that OpenVPN users need to be aware of when configuring their clients and/or servers, and how users address it depends on their needs (e.g. whether they want all DNS queries to go to the VPN's DNS servers, or whether they want only queries for certain domains to go to those servers).

---

@redongh -- please open a separate issue for support regarding the stub resolver method.

**Comment by tomeon** [comment]
I've closed this issue, but (as time permits) will look into whether the upstream issues still exist and, if not, will remove their mention from the README.
